README.md

```diff
@@ -29,7 +29,7 @@ LazyRow/LazyColumn for compose.
   chart.
 - [x] Month/Week headers and footers - Add headers/footers of any kind on each month/week.
 - [x] Easily scroll to any date/week/month on the calendar via swipe actions or programmatically.
-- [x] Use all RecyclerView/LazyRow/LazyColumn customizations since the calendar extends from
+- [x] Uses all RecyclerView/LazyRow/LazyColumn customizations since the calendar extends from
```

Comment:
The `use` here is the short form for `You can use all RecyclerView/LazyRow/LazyColumn....`, not saying that the library `uses` those APIs. Maybe we can phrase it differently, but `uses` would not be a sufficient replacement.